docs/ref-manual/cli/options.md

```diff
@@ -61,21 +61,28 @@ Adding a message makes it easier to track several runs. It is very useful if you
 To create the message above, we used  
 `certoraRun Bank.sol --verify Bank:Bank.spec --msg 'Removed an assertion'`
 
-### `--rule`
+### `--rule (alternatively: --rules)`
 
 **What does it do?**  
-Formally verifies a single property instead of the whole specification file. An invariant can also be selected.  
+Formally verifies one or more given properties instead of the whole specification file. An invariant can also be selected.  
 **When to use it?**  
-This option saves a lot of run time. Use it whenever you care about only a single property. The most common case is when you add a new rule to an existing specification. The other is when code changes cause a specific rule to fail; in the process of fixing the code, updating the rule, and understanding counterexamples, you likely want to verify only that specific rule.  
+This option saves a lot of run time. Use it whenever you care about only a specific subset of a specification's total properties. The most common case is when you add a new rule to an existing specification. The other is when code changes cause a specific rule to fail; in the process of fixing the code, updating the rule, and understanding counterexamples, you likely want to verify only that specific rule.  
 **Example**  
 If `Bank.spec` includes the following properties:  
 `invariant address_zero_cannot_become_an_account()`
 
 `rule withdraw_succeeds()`
+`rule withdraw_fails()`
 
 If we want to verify only `withdraw_succeeds`, we run  
 `certoraRun Bank.sol --verify Bank:Bank.spec --rule withdraw_succeeds`
 
+If we want to verify both `withdraw_succeeds` and `withdraw_fails`, we run  
+`certoraRun Bank.sol --verify Bank:Bank.spec --rule withdraw_succeeds withdraw_fails`
```

Comment:
I might use `--rules` here

Reply:
I didn't want to give the impression that the plural version (ie --rules) was somehow necessary in order to specify multiple rules. "--rules" is purely "syntactic sugar".